spec/lib/orcid/pub_mapper_spec.rb

```diff
@@ -151,6 +151,23 @@
     end
   end
 
+  context 'when year has whitespace' do
+    let(:pub_hash) do
+      base_pub_hash.dup.tap do |pub_hash|
+        pub_hash.delete(:date)
+        pub_hash[:year] = ' 1950'
+      end
+    end
+
+    it 'does not map' do
```

Comment:
should title be more like `maps with whitespace stripped`?

Reply:
Thanks for catching. I had originally had it not map then changed the functionality.